model_prices_and_context_window.json

```diff
@@ -7614,22 +7590,76 @@
         "litellm_provider": "voyage",
         "mode": "embedding"
     },
-    "voyage/voyage-lite-02-instruct": {
+    "voyage/voyage-finance-2": {
         "max_tokens": 4000,
         "max_input_tokens": 4000,
-        "input_cost_per_token": 0.0000001,
+        "input_cost_per_token": 0.00000012,
         "output_cost_per_token": 0.000000,
         "litellm_provider": "voyage",
         "mode": "embedding"
     },
-    "voyage/voyage-finance-2": {
-        "max_tokens": 4000,
-        "max_input_tokens": 4000,
+    "voyage/voyage-3-large": {
```

Comment:
can you submit these in a separate pr that just adds the new model pricing, without deleting any existing models 

@fzowl